### PR TITLE
Sample images set up

### LINKED ISSUE #29: Sample images and calibration files set up
- [x] #31
- [x] #32
- [x] #33
- [ ] #34
- [ ] #35
- [ ] #36

### LINKED ISSUE #31: Create a folder  for sample images

### PR DESCRIPTION
**Pull request by gyyyno1**
# Description

In path 'bubble_analyser/sample_images', sample images are imported for testing functions in the future.

Fixes #31 #32 

## Type of change

- [ ] Documentation (non-breaking change that adds or improves the documentation)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (non-breaking, back-end change that speeds up the code)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (whatever its nature)

## Key checklist

- [ ] All tests pass (eg. `python -m pytest`)
- [ ] The documentation builds and looks OK (eg. `python -m sphinx -b html docs docs/build`)
- [x] Pre-commit hooks run successfully (eg. `pre-commit run --all-files`)

## Further checks

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added or an issue has been opened to tackle that in the future. (Indicate issue here: # (issue))

### CODE REVIEW

File: poetry.lock
Comment (gyyyno1) [reply poetry.lock:1]: > This file should not be here. You're just adding images, so this one should not have been changed.
Just reverted the commit! :)

**Review by dalonsoa — APPROVED** [review]
All good!

**Comment by gyyyno1** [comment]
> All good!

Shall I merge sample_images to the main branch? I also have a branch working on calibration files and I want to move the folder to the 'test' folder in that branch (I should have do both of calibration and sample images within one branch...this is a bit unnecessary to have them in separate branches)

**Comment by dalonsoa** [comment]
Yeah, when things are closely related they can be tackled within the same branch with no problem. There's no need to overdo the git workflow.

And yes, you can merge this one, unless you want to wait for @dmesa16 approval.

**Review by dmesa16 — COMMENTED** [review]
Maybe we can change the names to Background, Ruler, and number the photos from 1 onwards

**Pull request merged**
